fiftyone/core/odm/__init__.py

```
@@ -49,6 +50,7 @@
     bulk_write,
 )
 from .dataset import (
+    ActiveFields,
```

Ruff (0.8.2)

`.dataset.ActiveFields` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)